### PR TITLE
Paramedic Lockers, Portafibs, and Chemvend Tweaks

### PR DESCRIPTION
**Pull request by rosieposieeee**
## About the PR

Added the content from my previous PRs on the original branch. As a recap of these PRs:
- [Portafibs](https://github.com/DeltaV-Station/Delta-v/pull/151) are smaller, portable defibs with less charge, for Paramedics to carry around to help people who JUST died. They hardly have enough charge to really bring back anyone who has been dead for a while, making them situational.
- [Paramedic access](https://github.com/DeltaV-Station/Delta-v/pull/197) was actually added, which means the paramedic secure windoor now works. Oh, and doctors can't nab the really valuable Paramedic items like the hud and suit and make paramedics into shittier doctors with no chem access.
- Added more of each jug to the [chemvend](https://github.com/DeltaV-Station/Delta-v/pull/199) so that restocks will be happening less constantly.

On top of that, I made additional tweaks and fixes to those previous PRs.

- Portafibs were only in duffels by accident, fixed that
- added the emagged chem dispenser chemicals to the chemvend since the dispenser will not be around very often
- gave paramed access for doctors when there's a skeleton crew
- I also made the rollerbed inside paramedic gear the emergency one since it seems more fitting, stands out more, is less ugly, and is otherwise literally identical.

https://user-images.githubusercontent.com/52761126/263178098-72f4e837-b8f3-48c7-b287-cfaf527f6178.mp4

image

image

Tested everything again and it all works.

## Technical details

I had to add napalm and ultravasculine bottles to our own code.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

File: Resources/Prototypes/Access/misc.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@
   - Brig
   - Engineering
   - Medical
+  - Paramedic
```
Comment (FluffiestFloof) [inline Resources/Prototypes/Access/misc.yml:18]: add comment `// DeltaV - Add Paramedic access`

File: Resources/Prototypes/Access/medical.yml [hunk 2]
```diff
@@ -20,3 +20,4 @@
   - ChiefMedicalOfficer
   - Medical
   - Chemistry
+  - Paramedic
```
Comment (FluffiestFloof) [inline Resources/Prototypes/Access/medical.yml:23]: add comment `// DeltaV - Add Paramedic access`

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/medical.yml [hunk 3]
```diff
@@ -7,3 +7,4 @@
     Bloodpack: 5
     EpinephrineChemistryBottle: 3
     Syringe: 5
+    Portafib: 1
```
Comment (FluffiestFloof) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/medical.yml:10]: add comment `// DeltaV - Add Portafibs`
or `// DeltaV - Add Portafibs, see Prototypes/DeltaV/Entities/Objects/Devices/Medical/portafib.yml`

File: Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/backpack.yml [hunk 4]
```diff
@@ -28,3 +28,4 @@
       - id: BoxSurvivalMedical
       - id: CheapRollerBedSpawnFolded
       - id: BodyBag_Folded
+      - id: Portafib
```
Comment (FluffiestFloof) [inline Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/backpack.yml:31]: add comment `// DeltaV - Add Portafibs`
or `// DeltaV - Add Portafibs, see Prototypes/DeltaV/Entities/Objects/Devices/Medical/portafib.yml`

File: Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/duffelbag.yml [hunk 5]
```diff
@@ -8,3 +8,4 @@
       - id: BoxSurvivalMedical
       - id: CheapRollerBedSpawnFolded
       - id: BodyBag_Folded
+      - id: Portafib
```
Comment (FluffiestFloof) [inline Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/duffelbag.yml:11]: add comment `// DeltaV - Add Portafibs`
or `// DeltaV - Add Portafibs, see Prototypes/DeltaV/Entities/Objects/Devices/Medical/portafib.yml`

File: Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/satchel.yml [hunk 6]
```diff
@@ -8,3 +8,4 @@
       - id: BoxSurvivalMedical
       - id: CheapRollerBedSpawnFolded
       - id: BodyBag_Folded
+      - id: Portafib
```
Comment (FluffiestFloof) [inline Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/satchel.yml:11]: add comment `// DeltaV - Add Portafibs`
or `// DeltaV - Add Portafibs, see Prototypes/DeltaV/Entities/Objects/Devices/Medical/portafib.yml`

File: Resources/Prototypes/Entities/Structures/Storage/Closets/Lockers/lockers.yml [hunk 7]
```diff
@@ -223,7 +223,7 @@
     stateDoorOpen: paramed_open
     stateDoorClosed: paramed_door
   - type: AccessReader
-    access: [ [ "Medical" ] ]
+    access: [ [ "Paramedic" ] ]
```
Comment (FluffiestFloof) [inline Resources/Prototypes/Entities/Structures/Storage/Closets/Lockers/lockers.yml:226]: add comment `// DeltaV - Change Medical access to Paramedic`

File: Resources/Prototypes/Roles/Jobs/Medical/paramedic.yml [hunk 8]
```diff
@@ -14,6 +14,7 @@
   supervisors: job-supervisors-cmo
   access:
   - Medical
+  - Paramedic
```
Comment (FluffiestFloof) [inline Resources/Prototypes/Roles/Jobs/Medical/paramedic.yml:17]: add comment `// DeltaV - Add Paramedic access`

File: Resources/Prototypes/Roles/Jobs/Medical/medical_doctor.yml [hunk 9]
```diff
@@ -15,6 +15,7 @@
   - Maintenance
   extendedAccess:
   - Chemistry
+  - Paramedic
```
Comment (FluffiestFloof) [inline Resources/Prototypes/Roles/Jobs/Medical/medical_doctor.yml:18]: add comment `// DeltaV - Add Paramedic access`

File: Resources/Prototypes/Roles/Jobs/Medical/chief_medical_officer.yml [hunk 10]
```diff
@@ -25,6 +25,7 @@
   canBeAntag: false
   access:
   - Medical
+  - Paramedic
```
Comment (FluffiestFloof) [inline Resources/Prototypes/Roles/Jobs/Medical/chief_medical_officer.yml:28]: add comment `// DeltaV - Add Paramedic access`

**Review by FluffiestFloof — COMMENTED** [review]
I love comments.

They are helpful in figuring what is from especially when touching the base game files

File: Resources/Prototypes/Access/misc.yml (hunk 1 above)
Comment (rosieposieeee) [reply Resources/Prototypes/Access/misc.yml]: added all these

**Review by DebugOk — CHANGES_REQUESTED** [review]
Paramedic is not defined within [Content.Shared/Access/Components/IdCardConsoleComponent.cs](https://github.com/DeltaV-Station/Delta-v-rebase/blob/master/Content.Shared/Access/Components/IdCardConsoleComponent.cs#L45) meaning its an unobtainable access level outside of spawning with it.

**Comment by rosieposieeee** [comment]
> Paramedic is not defined within [Content.Shared/Access/Components/IdCardConsoleComponent.cs](https://github.com/DeltaV-Station/Delta-v-rebase/blob/master/Content.Shared/Access/Components/IdCardConsoleComponent.cs#L45) meaning its an unobtainable access level outside of spawning with it.

image
Done, whoops. Never even been hop so I didn't think about it. Will fix merge conflicts in a sec.

**Review by DebugOk — CHANGES_REQUESTED** [review]
Test failures are valid. Chemvend changes introduce free money for cargo.

```A total of 1 test files matched the specified pattern.
  Failed NoCargoOrderArbitrage [1 s]
  Error Message:
     Found arbitrage on CrateVendingMachineRestockChemVend cargo product! Cost is 3820 but sell is 6770!
  Expected: less than or equal to 3820
  But was:  6770.0d```

File: Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/backpack.yml (hunk 4 above)
Comment (Colin-Tel) [reply Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/backpack.yml:31]: There's no need to add comments here indicating a Delta V change, when the entire file is a Delta V change 😉

**Review by Colin-Tel — DISMISSED** [review]
Looks good! We'll just have to see how this affects the balance of things.

File: Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/backpack.yml (hunk 4 above)
Comment (FluffiestFloof) [reply Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/backpack.yml:31]: _Looks at date_
Hm yeah probably shouldn't review right after I wake up

**Comment by Adrian16199** [comment]
Not to be chuggin logs with a meaningless comment but..... I absolutely love this PR. Atleast as a paramedic enjoyer. Being able to carry defibs without it usin half your bag at the cost of power, definitely 10/10 because consequences if you dont pay attention to it as balance, givin paramedics additional access durin skeleton crew, absolutely 10/10 because i want to be able to paramed but also get chems if im alone at medical without CMO.

**Review by Colin-Tel — DISMISSED** [review]
*me saying the same thing post-merge fix*
Looks good! We'll just have to see how this affects the balance of things.

**Comment by DebugOk** [comment]
Fuck it

**Comment by DebugOk** [comment]
The fuck you mean merging failed

**Pull request merged**